guides/rql.md

```diff
@@ -117,20 +117,21 @@ WHERE item.counter IN (1,2,3)
 
 #### item\_occurrence
 
-`item_occurrence` is a table where each row contains data about a single
-occurrence, as well as the item that the occurrence is associated with.
-Column names starting with "item." reference the item, and all other
-column names reference the occurrence. Column names that do not exist in
-a particular occurrence evaluate to NULL.
+`item_occurrence` is a table where each row contains data for a single
+occurrence and the item it is associated with. Column names starting with "item."
+reference the item, and all other column names reference the occurrence.
+Column names that do not exist in a particular occurrence evaluate to NULL.
+
+The following columns exist for every row in `item\_occurrence`:
```

Comment:
ah, yep, I was copying the header where the `\` is needed, but apparently not needed here. thanks!